lab.R

```diff
@@ -187,22 +235,36 @@ mob_df = read_csv(file.path('data', 'mobility.csv'))
 #' 
 
 #' 1. *This is just one way we could get at the relationship between stay-at-home in June and the peak of the outbreak in July.  What are some other approaches we might take?*
-#' 
+#' We can also investigate the relationship between mobility data for other places (i.e. retial) and the number of cases. 
 #' 
 #' 
 
 #' 2. *Construct a dataframe `parks_june` that reports the mean level of "parks" mobility for each county in June 2020.  Call the variable `parks`.  (Just so the automatic checks know where to look.)  Note that the Google data has a lot of gaps for the `parks` type.  Use the `na.rm = TRUE` argument in `mean()` to handle missing values, and then filter out missing values.  The final dataframe should have three columns: county name, FIPS code, and `parks`.  And it should have one row for each county in the mobility data for which we have an estimate for "parks".*
-#' 
+parks_june = mob_df %>% 
+    filter(type == "parks") %>% 
+    filter(date >= '2020-06-1' & date <= '2020-06-30') %>% 
+    group_by(county, fips) %>% 
+    summarise(parks=mean(pct_diff, na.rm=TRUE)) %>% 
+    filter(!is.na(parks))
+    
+    
 
 #' 3. *Construct a dataframe `cases_july` that reports the total level of new cases per 1 million residents of each county in July 2020.  (Don't worry about negative values.  I'm just asking you to do `sum(cases_per_pop)`.)  This dataframe should have three columns and one row for each county in the Covid-19 data.*
-#' 
+cases_july =  full_join(nytcovcounty, pop) %>% 
```

Comment:
We want to use `covid_df` here, which is already filtered down to California counties and has the case and death counts per pop.